R/add_dd.R

```diff
@@ -0,0 +1,304 @@
+#' Add Delay Discharge to working file
+#'
+#' @param data The input data frame
+#' @param year The year being processed
+#'
+#' @return A data frame linking delay discharge cohorts
+#' @export
+#'
+#' @family episode file
+add_dd <- function(data, year) {
+  year_param <- year
+
+  data <- data %>%
+    dplyr::mutate(
+      # remember to revoke the cij_end_date with dummy_cij_end
+      cij_start_date_lower = cij_start_date - lubridate::days(1),
```

Comment:
Run `lintr::lint("R/add_dd.R")` to see others like this that need fixing.
```suggestion
      cij_start_date_lower = .data$cij_start_date - lubridate::days(1),
```